### DISCUSSION TITLE
Add `report` feature - for reporting malware packages

### DISCUSSION THREAD

**Discussion by srmish-jfrog** [npm/feedback#651] [CLI] [answered]
It would be extremely convenient if the npm CLI would allow reporting packages as malware, instead of going through the Web UI.
This will make reporting malware packages in batch much easier (instead of going through the website).
Perhaps the various fields (Subject/Package/Version/Description) could be given via an input JSON file

**Comment by MylesBorins** [comment, marked as answer]
Hey @srmish-jfrog, great suggestion. For changes to the CLI the appropriate next step is to open an issue over at [the npm CLI RFC repo](https://github.com/npm/rfcs) to suggest the change. While there will be coordination with the back end team to determine how to support this we should confirm support in the CLI is desired by the team first.
